lib/screens/onboarding/continue_recovery.dart

```diff
@@ -313,26 +298,26 @@ class _ContinueRecoveryState extends State<ContinueRecovery> {
       recovering = true;
       canClaim = true;
     });
-    
-    String accountName = SettingsNotifier.of(context, listen: false).accountName;
-    
+
+    String accountName =
+        SettingsNotifier.of(context, listen: false).accountName;
+
     try {
+      await EosService.of(context, listen: false)
```

Comment:
We need to kind of use the same code formatting otherwise there's lots of uneccessary "changes" like this